test/make-large-log

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
```

Comment:
We aren't parsing logs but processing in realtime and stuffing the data in the DB so this script can go away